composer.json

```diff
@@ -13,8 +13,9 @@
     "php": ">=5.6"
   },
   "require-dev": {
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
```

Comment:
This isn't currently needed because we define the included standards here:

https://github.com/WordPress/two-factor/blob/1c828f90ee9461a3577c35d461d6160e93d336a6/phpcs.xml#L5